<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>714</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>536</width>
    <height>714</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>536</width>
    <height>714</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginWidget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>572</y>
     <width>440</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>440</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>440</width>
     <height>70</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>66</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="login_button">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>登  录</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>58</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="register_button">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>注  册</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>65</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>415</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>415</width>
     <height>111</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>415</width>
     <height>111</height>
    </size>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>9</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>账号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>72</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="account_edit">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>9</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>30</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>账号/手机</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwd_edit">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>72</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>30</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>6-15位数字、字母组合</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="img_lbl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="forget_lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>680</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>忘记账号密码?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
